extendeddatasquare_test.go

```diff
@@ -188,14 +232,15 @@ func BenchmarkExtensionEncoding(b *testing.B) {
 // BenchmarkExtension benchmarks extending datasquares sizes 4-128 using all
 // supported codecs (both encoding and root computation)
 func BenchmarkExtensionWithRoots(b *testing.B) {
+	chunkSize := 256
```

Comment:
this appears to have been set to share size, if that's the case then perhaps we should update these to 512

Reply:
great idea! Created https://github.com/celestiaorg/rsmt2d/issues/237